config/config.go

```diff
@@ -15,13 +15,15 @@ type Config struct {
 	Randomize     bool        `yaml:"randomize"`
 	StopOnFailure bool        `yaml:"stop_on_failure"`
 	FeaturesPaths []string    `yaml:"features_path"`
+	StubsPath     string      `yaml:"stubs_path"`
 	Resources     []*Resource `yaml:"resources"`
 }
 
 type Resource struct {
 	Name   string            `yaml:"name"`
 	Type   string            `yaml:"type"`
 	Params map[string]string `yaml:"params"`
+	Stubs  map[string][]byte // static stubs available to all resources
```

Comment:
I'm not a fan of this, I think config should act as a user configuration, rather than parameters, beside not all resources need this

Reply:
Agreed, adjusted